Project2015To2017.Console/Project2015To2017.Console.csproj

```diff
@@ -15,22 +14,29 @@
 		<Version>3.0.0</Version>
 		<OutputType>Exe</OutputType>
 		<AssemblyVersion>3.0.0.0</AssemblyVersion>
+		<RestoreAdditionalProjectSources>
+			$(RestoreAdditionalProjectSources);
+			https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+		</RestoreAdditionalProjectSources>
 	</PropertyGroup>
 
 	<PropertyGroup Condition="$(Pack) != 'true'">
 		<TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
 	</PropertyGroup>
 
 	<PropertyGroup Condition="$(Pack) == 'true'">
-		<ToolCommandName>csproj-to-2017</ToolCommandName>
+		<ToolCommandName>dotnet-migrate-2017</ToolCommandName>
```

Comment:
I think we should be doing this in a new separate console project, I don’t want to kill the current CLI command because people might be upgrading and that’s very unexpected.

Reply:
And keep it with buggy CLI? If we define this to be 4.0.0 (I expected 3.0.0 - major version increase after this and a couple more PRs, not so early), we can introduce this breaking change and keep it this way (drop old CLI), but specify it in release notes (I mean, write real change notes that specify changes in behaviors and such).

Reply:
I mean, no problem, easily done, but should we really keep old CLI?